tests/windows_test.go

```diff
@@ -255,6 +259,77 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", func() {
 					`DNSDomain[\n\r\t ]+`+expectedSearchDomain+`[\n\r\t ]+`)
 			}, 360)
 		})
+
+		Context("VMI with bridge binding is created", func() {
+			var winVmiIp string
+			var winPodIp string
+
+			BeforeEach(func() {
+				windowsVMI.Spec.Domain.Devices.Interfaces = []v1.Interface{
+					{
+						Name: "default",
+						InterfaceBindingMethod: v1.InterfaceBindingMethod{
+							Bridge: &v1.InterfaceBridge{},
+						},
+					},
+				}
+
+				By("Starting Windows VirtualMachineInstance with bridge binding")
+				windowsVMI, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(windowsVMI)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStartWithTimeout(windowsVMI, 360)
+
+				cli = winrnLoginCommand(virtClient, windowsVMI)
+			})
+
+			It("should have ip value equal to matching pod ip", func() {
+				windowsVMI, err = virtClient.VirtualMachineInstance(windowsVMI.Namespace).Get(windowsVMI.Name, &metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				Eventually(func() int {
+					return len(windowsVMI.Status.Interfaces)
+				}, time.Minute*1, time.Second*15).Should(BeNumerically("==", 1))
```

Comment:
I think you need to return here an IP address and the condition should be that it is the right one.

As it stands now, you assume that once an interface is detected (by the guest-agent I guess), it also includes an IP address. But this does not have to be the case. The interface will probably also be there and reported immediately, but the IP address may take more time to appear.